src/overlays/actors/ovl_En_Gm/z_en_gm.c

```diff
@@ -7,8 +14,27 @@
 void EnGm_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnGm_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnGm_Update(Actor* thisx, GlobalContext* globalCtx);
+void EnGm_DrawProc(Actor* thisx, GlobalContext* globalCtx);
+
+void func_80A3D838(EnGm* this, GlobalContext* globalCtx);
+void func_80A3DFBC(EnGm* this, GlobalContext* globalCtx);
+void func_80A3DB04(EnGm* this, GlobalContext* globalCtx);
+void func_80A3DC44(EnGm* this, GlobalContext* globalCtx);
+void func_80A3DBF4(EnGm* this, GlobalContext* globalCtx);
+void func_80A3DD7C(EnGm* this, GlobalContext* globalCtx);
+void EnGm_ProcessChoiceIndex(EnGm* this, GlobalContext* globalCtx);
+void func_80A3DF00(EnGm* this, GlobalContext* globalCtx);
+void func_80A3DF60(EnGm* this, GlobalContext* globalCtx);
+void EnGm_UpdateEye(EnGm* this);
+void EnGm_SetTextID(EnGm* this);
+void func_80A3E090(EnGm* this);
+
+s32 func_80A3D7C8(void);
+
```

Comment:
```suggestion
void func_80A3DF60(EnGm* this, GlobalContext* globalCtx);

```
those prototypes aren't needed, the functions are defined before being used